.azure-pipelines/warden.yml

```diff
@@ -72,7 +72,7 @@ jobs:
   dependsOn: 'Build'
 
   pool:
-    vmImage: 'ubuntu-16.04'  
+    vmImage: 'ubuntu-20.04'
```

Comment:
Did you choose 20.04 over 18.04 for a particular reason?

Reply:
We should consider sticking to 18.04 until we attempt to move everything to 20.04. However if this is the start of that attempt I'm fine with what you have.

Reply:
I think it might have been because of the comment @kurtzeborn made on the linked issue.

Reply:
@mitchdenny Yes it due to Scotts comment. Should I go ahead with 20.0.4?

Reply:
I'd probably go with 18.04 as its our primary Linux image right now unless there was an explicit reason for going 20.04.